frontend/src/metabase/querying/notebook/components/JoinStep/JoinTablePicker/JoinTablePicker.tsx

```diff
@@ -92,6 +92,6 @@ const CONTAINER_STYLE = {
 
 const RIGHT_CONTAINER_STYLE = {
   width: 37,
-  height: 37,
+  height: "100%",
```

Comment:
:x: We need to change the height of the button, not its container.
Otherwise not the entire "blue area" is interactive.

![image](https://github.com/user-attachments/assets/2f3eb877-57e0-49b1-bc3b-1a7cfce5174f)

Reply:
And let's fix this issue for data source as well

![image](https://github.com/user-attachments/assets/d3e960a7-2d39-4bd7-aa37-094f5c062ab6)

Reply:
Please pay attention to the icon on left-hand side (both cases) - it needs to be vertically centered, and text has to go on the right of it

Reply:
> We need to change the height of the button, not its container.

Good catch. Off course the whole button had a `cursor: pointer` applied so it looked like it worked. Fixed it in 1577a79bc6d15ff2f79bb70b4bca4bb41cd37610

> Please pay attention to the icon on left-hand side (both cases) - it needs to be vertically centered, and text has to go on the right of it

It already does (in this branch) on all browsers on my machine.
<img width="430" alt="Screenshot 2024-11-16 at 00 25 52" src="https://github.com/user-attachments/assets/a0f7d42e-456e-4c4a-b835-e21e8b7f3080">